e3sm_diags/derivations/acme.py

```diff
@@ -159,7 +159,7 @@ def precst(precc, precl):
 
 def tauxy(taux, tauy):
     """tauxy = (taux^2 + tauy^2)sqrt"""
-    var = (taux ** 2 + tauy ** 2) ** 0.5
+    var = (taux**2 + tauy**2) ** 0.5
```

Comment:
The latest version of `black` (22.1.0) now fixes unnecessary spaces between `**`